app/model/trainer.py

```
@@ -28,7 +41,14 @@ def build_interactions(dataset, interaction_df, user_brand_df, brand_df):
     real = list(zip(interaction_df["user_id"], interaction_df["brand_id"], interaction_df["weight"]))
     return dataset.build_interactions(real + dummy_interactions)
 
-def train_model(interactions, weights, user_features):
-    model = LightFM(loss="warp")
-    model.fit(interactions, sample_weight=weights, user_features=user_features, epochs=10, num_threads=2)
+def train_model(interactions, weights, user_features, item_features):
+    from lightfm import LightFM
```

Ruff (0.12.2)

Redefinition of unused `LightFM` from line 1

Remove definition: `LightFM`

(F811)